src/apps/recommended-material/RecommendedMaterial.tsx

```diff
@@ -77,6 +80,13 @@ const RecommendedMaterial: React.FC<RecommendedMaterialProps> = ({
     );
   };
 
+  const trackData = () =>
+    track("click", {
```

Comment:
I’m uncertain whether this should be a `"link"`. I’ll check with Tue before making any changes.